api/read.rb

```diff
@@ -0,0 +1,4 @@
+post "#{APIPREFIX}/users/:user_id/read" do |user_id|
```

Comment:
@robrap The baseline change is required against users. So I thought its a good idea to keep 'read' endpoint against user than to categorising it as `thread read`.

I first thought to add it to users.rb; but then looking at other such flagging endpoints being split into files (e.g. flag, subscribe), I just tried to follow convention there.

Is it bad?